src/views/forms/AccountManagementForms/DepositForm/DepositButtonAndReceipt.tsx

```diff
@@ -108,20 +108,6 @@ export const DepositButtonAndReceipt = ({
 
   const totalFees = (summary?.bridgeFee || 0) + (summary?.gasFee || 0);
 
-  const { toAmount, toAmountMin } = useMemo(() => {
-    if (isCctp) {
-      return {
-        toAmount: summary?.toAmount,
-        toAmountMin: summary?.toAmountMin,
-      };
-    } else {
-      return {
-        toAmount: summary?.toAmount && formatUnits(BigInt(summary.toAmount), usdcDecimals),
```

Comment:
No longer need to do this, because abacus was not normalizing summary?.toAmount correctly before, but is fixed now.